web_src/css/markup/content.css

```diff
@@ -1,6 +1,6 @@
 .markup {
   overflow: hidden;
-  font-size: 16px;
+  font-size: 14px;
```

Comment:
Maybe we should remove this and use the parent's font-size.

Body's default font-size is also 14px IIRC.